plugin/src/main/java/org/opensearch/ml/action/deploy/TransportDeployModelAction.java

```diff
@@ -176,17 +176,21 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<MLDepl
                                 deployModel(deployModelRequest, mlModel, modelId, wrappedListener, listener);
                             }
                         }, e -> {
-                            log.error("Failed to Validate Access for ModelId " + modelId, e);
+                            if (isHidden) {
+                                log.error("Failed to Validate Access for the given model", e);
+                            } else {
+                                log.error("Failed to Validate Access for ModelId " + modelId, e);
+                            }
                             wrappedListener.onFailure(e);
```

Comment:
For deploy model api, you removed the model id by "the given model". What about update model api, should we also remove the model id?
`if (isSuperAdmin) {
                                updateRemoteOrTextEmbeddingModel(modelId, updateModelInput, mlModel, user, wrappedListener);
                            } else {
                                wrappedListener
                                    .onFailure(
                                        new OpenSearchStatusException(
                                            "User doesn't have privilege to perform this operation on this model, model ID " + modelId,
                                            RestStatus.FORBIDDEN
                                        )
                                    );
                            }`